packages/messaging-api-line/src/LineClient.ts

```diff
@@ -1630,6 +1630,50 @@ export default class LineClient {
       .then((res) => res.data, handleError);
   }
 
+  /**
+   * Get Group Summary
+   *
+   * Gets the group ID, group name, and group icon URL of a group where the LINE Official Account is a member.
+   *
+   * [Official document - get group summary](https://developers.line.biz/en/reference/messaging-api/#get-group-summary)
+   *
+   * @param groupId - Group ID. Found in the `source` object of [webhook event objects](https://developers.line.biz/en/reference/messaging-api/#webhook-event-objects).
+   * @returns Returns status code `200` and a JSON object with the following information.
+   *
+   * groupId:
+   * - Group ID
+   *
+   * groupName:
+   * - Group name
+   *
+   * pictureUrl:
+   * - Group icon URL
+   */
+  getGroupSummary(groupId: string): Promise<Types.Group> {
+    return this._axios
+      .get(`/v2/bot/group/${groupId}/summary`)
+      .then((res) => res.data, handleError);
+  }
+
+  /**
+   * Get Members In Group Count
+   *
+   * Gets the count of members in a group. You can get the member in group count even if the user hasn't added the LINE Official Account as a friend or has blocked the LINE Official Account.
+   *
+   * [Official document - get members in group count](https://developers.line.biz/en/reference/messaging-api/#get-members-group-count)
+   *
+   * @param groupId - Group ID. Found in the `source` object of [webhook event objects](https://developers.line.biz/en/reference/messaging-api/#webhook-event-objects).
+   * @returns Returns status code `200` and a number representing group member count.
+   *
+   * count:
+   * - The count of members in the group. The number returned excludes the LINE Official Account.
+   */
+  getGroupMembersCount(groupId: string): Promise<Types.MemberCount> {
+    return this._axios
```

Comment:
@hch903 Please rebase the master branch and rename `this._axios` to `this.axios`.